Source/Core/Common/ChunkFile.h

```diff
@@ -330,26 +335,26 @@ class PointerWrap
 
   DOLPHIN_FORCE_INLINE void DoVoid(void* data, u32 size)
   {
-    if (mode != MODE_MEASURE && (*m_ptr_current + size) > m_ptr_end)
+    if (!IsMeasureMode() && (*m_ptr_current + size) > m_ptr_end)
     {
       // trying to read/write past the end of the buffer, prevent this
-      mode = MODE_MEASURE;
+      SetMeasureMode();
```

Comment:
I squashed most of the prior commits.

This pair of changes is new in the squash. I had previously omitted them because I assumed the inline declaration meant it was in an especially critical path and I wanted to avoid adding any function calls, but according to https://github.com/dolphin-emu/dolphin/pull/892/commits that wasn't the case.